MonoGame.Framework/Graphics/Metrics.cs

```diff
@@ -0,0 +1,56 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace Microsoft.Xna.Framework.Graphics
+{
+    /// <summary>
+    /// The rendering information from <see cref="GraphicsDevice"/>. 
+    /// Useful for debugging and profiling purposes. 
+    /// Resetted every frame after draw at <see cref="GraphicsDevice.Present"/>.
+    /// </summary>
+    public struct Metrics
+    {
+        internal ulong _batchCount;
+        internal ulong _drawCount;
+        internal ulong _primitivesCount;
+
+        /// <summary>
+        /// The count of batches sended to <see cref="SpriteBatch"/>.
+        /// </summary>
+        public ulong BatchCount { get { return _batchCount; } }
+
+        /// <summary>
+        /// The count of draw calls.
+        /// </summary>
+        public ulong DrawCount { get { return _drawCount; } }
+
+        /// <summary>
+        /// The count of rendered primitives.
+        /// </summary>
+        public ulong PrimitivesCount { get { return _primitivesCount; } }
+        
+        internal void Reset()
```

Comment:
I tested - because Metrics is a struct and property of GraphicsDevice a call from Draw such as

> GraphicsDevice.Metrics.Reset() does not have effect